src/games/supported/LostLuggage.hpp

```diff
@@ -0,0 +1,63 @@
+/* *****************************************************************************
+ * A.L.E (Arcade Learning Environment)
+ * Copyright (c) 2009-2013 by Yavar Naddaf, Joel Veness, Marc G. Bellemare and 
+ *   the Reinforcement Learning and Artificial Intelligence Laboratory
+ * Released under the GNU General Public License; see License.txt for details. 
+ *
+ * Based on: Stella  --  "An Atari 2600 VCS Emulator"
+ * Copyright (c) 1995-2007 by Bradford W. Mott and the Stella team
+ *
+ * *****************************************************************************
+ */
+#ifndef __LOSTLUGGAGE_HPP__
+#define __LOSTLUGGAGE_HPP__
+
+#include "../RomSettings.hpp"
+
+/* RL wrapper for Lost Luggage */
+class LostLuggageSettings : public RomSettings {

```

Comment:
'Fire' resets the game, so you should also implement isLegal for this game. See Skiing.{hpp,cpp} for details.